relayer/relays/beacon/header/header.go

```diff
@@ -386,3 +457,40 @@ func (h *Header) isInitialSyncPeriod() bool {
 	lastFinalizedPeriod := h.syncer.ComputeSyncPeriodAtSlot(h.cache.Finalized.LastSyncedSlot)
 	return initialPeriod == lastFinalizedPeriod
 }
+
+func (h *Header) findClosestCheckPoint(slot uint64) (state.FinalizedHeader, error) {
```

Comment:
IIUC it's not closest anymore instead the most distant. So the previous name `findCheckPointBackward` make more sense here?

Reply:
Renamed here https://github.com/Snowfork/snowbridge/pull/1156/commits/b5334dc6aeb077546b47bb722f87c073256281b8.